app/src/main/java/com/example/atheneum/viewmodels/NotificationsViewModel.java

```diff
@@ -0,0 +1,78 @@
+package com.example.atheneum.viewmodels;
+
+import android.arch.core.util.Function;
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.Transformations;
+import android.arch.lifecycle.ViewModel;
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import com.example.atheneum.models.Notification;
+import com.example.atheneum.utils.FirebaseQueryLiveData;
+import com.example.atheneum.viewmodels.FirebaseRefUtils.DatabaseWriteHelper;
+import com.example.atheneum.viewmodels.FirebaseRefUtils.NotificationsRefUtils;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.Query;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NotificationsViewModel extends ViewModel {
+    // Raw stream of read-only DataSnapshot values retrieved from query
+    private final FirebaseQueryLiveData queryLiveData;
+    private final LiveData<List<Notification>> notificationLiveData;
+    // Notifications query for firebase
+    private final Query notificationsRef;
+
+    private final String TAG = NotificationsViewModel.class.getSimpleName();
+
+    public NotificationsViewModel(String userID) {
+        notificationsRef = NotificationsRefUtils.getNotificationsRef(userID);
+        queryLiveData = new FirebaseQueryLiveData(notificationsRef);
+        notificationLiveData = Transformations.map(queryLiveData, new Function<DataSnapshot, List<Notification>>() {
+            @Override
+            public List<Notification> apply(DataSnapshot dataSnapshot) {
+                List<Notification> notificationList = new ArrayList<Notification>();
+                for (DataSnapshot data : dataSnapshot.getChildren()) {
+                    // reverses elements for reverse-chronological time ordering in list
+                    notificationList.add(0, data.getValue(Notification.class));
+                }
+                return notificationList;
+            }
+        });
+    }
+
+    /**
+     * Make notification seen
+     *
+     * @param notification
+     */
+    public void makeNotificationSeen(Notification notification) {
+        if (!notification.getIsSeen()) {
+            notification.setIsSeen(true);
+            DatabaseWriteHelper.makeNotificationSeen(notification);
+        }
+    }
+
+    /**
+     * Delete notification
+     *
+     * @param position
+     */
+    public void deleteNotification(int position) {
+        List<Notification> notifications = getNotificationLiveData().getValue();
```

Comment:
Wait, since you're storing the userID in the viewmodel and the notification has a notificationID which is unique, who not pass in a notification object to this method directly? I think that would be cleaner than trying to delete it by position. Also, Firebase listens for changes to the list of notifications for the list of users so this isn't really necessary since the LiveData ensures that the syncing happens properly on delete.